style/layer/aeroway.js

```diff
@@ -0,0 +1,250 @@
+"use strict";
+
+import * as Color from "../constants/color.js";
+
+const name_en = [
+  "coalesce",
+  ["get", "name:en"],
+  ["get", "name:latin"],
+  ["get", "name"],
+];
+
+const minorAirport = [
+  "any",
+  ["!", ["has", "iata"]],
+  ["!", ["has", "icao"]],
+  ["in", ["get", "class"], ["literal", ["private"]]],
+];
+
+const iconLayout = {
+  "icon-image": [
+    "match",
+    ["get", "class"],
+    "military",
+    "military_airport",
+    "airport",
+  ],
+  "text-anchor": "bottom",
+  "text-variable-anchor": [
+    "bottom",
+    "bottom-right",
+    "bottom-left",
+    "right",
+    "left",
+  ],
+  "text-padding": 8,
+  "icon-allow-overlap": false,
+};
+
+export const fill = {
+  id: "airport_fill",
+  type: "fill",
+  filter: ["==", "class", "aerodrome"],
+  paint: {
+    "fill-color": Color.airportFill,
+  },
+  layout: {
+    visibility: "visible",
+  },
+  source: "openmaptiles",
+  metadata: {},
+  "source-layer": "aeroway",
+};
+
+export const outline = {
+  id: "airport_outline",
+  type: "line",
+  filter: ["==", "class", "aerodrome"],
+  paint: {
+    "line-color": Color.airportOutline,
+  },
+  layout: {
+    visibility: "visible",
+  },
+  source: "openmaptiles",
+  metadata: {},
+  "source-layer": "aeroway",
+};
+
+export const runway = {
+  id: "airport_runway",
+  type: "line",
+  filter: ["all", ["==", "class", "runway"], ["==", "$type", "LineString"]],
+  paint: {
+    "line-color": Color.airportRunway,
+    "line-width": {
+      base: 1.7,
+      stops: [
+        [12, 3],
+        [15, 15],
+        [17, 30],
+      ],
+    },
+  },
+  layout: {
+    "line-cap": "round",
```

Comment:
Try using `square` instead of `round` on line caps for runways and taxiways. Since these features tend to have square ends in real life, I think they look a little better this way.